controllers/drpolicy_controller.go

```diff
@@ -88,10 +88,17 @@ func (r *DRPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 			return ctrl.Result{}, fmt.Errorf("validate: %w", u.validatedSetFalse(reason, err))
 		}
 
+		// TODO: New condition type is needed for clusters deploy and fencing
+		// handled after this function.
 		if err := drClustersDeploy(drpolicy, &manifestWorkUtil, &ramenConfig); err != nil {
 			return ctrl.Result{}, fmt.Errorf("drclusters deploy: %w", u.validatedSetFalse("DrClustersDeployFailed", err))
 		}
```

Comment:
If we pass this function, we may need to update an in-mem version of the status, so that an update at line 99 reports validated as true and fenced as false.